db/memtable_list.cc

```diff
@@ -736,17 +736,24 @@ void MemTableList::RemoveOldMemTables(uint64_t log_number,
   assert(to_delete != nullptr);
   InstallNewVersion();
   auto& memlist = current_->memlist_;
+  std::list<MemTable*> to_delete_memlist;
```

Comment:
Can we use `autovector` defined in `util/autovector.h`?

Reply:
Yes, autovector can be use here.